src/TMLeague/wwwroot/data/leagues/loiaf/seasons/s21/divisions/q1.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "Qualifiers Group 1",
+  "judge": null,
+  "players": [
+    "spartan109",
+    "Sean_Snow",
+    "jcggarrett1",
+    "Turambar",
+    "oAntares",
+    "diamonnd",
+    "s1naps3"
+  ],
+  "games": [
```

Comment:
Games `284867, 285113, 285118, 285160, 285292, 285543` not exists.

Reply:
Yeah like you thought, these games were aborted. As we discussed, I'll try to make a fake game file with the results from the end state and we'll see if that works.